### PR TITLE
Added functionality to the core of taglog, logship and logtag commands to support non-@tag fields

### PR DESCRIPTION
**Pull request by danielabel**
Nick please review.

### CODE REVIEW

**Comment by danielabel** [comment]
On hold, fixing build failure and removing some unused code that was missed

**Comment by danielabel** [comment]
Should be better now.

File: tagalog/__init__.py [hunk 1]
```diff
@@ -26,6 +26,33 @@ def stamp(iterable, key='@timestamp'):
...
+
+def _process_fields(item, fields):
+    item.update(fields)
+    return item
+
+def _prepare_fields(fields):
+    prepared_fields = {}
+
+    for field in fields:
+        field_name, _, field_value =  field.partition('=')
```
Comment (nickstenning) [inline tagalog/__init__.py]: Would prefer `field_name, field_value = field.split('=', 1)`

File: test/integration/test_commandline.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from __future__ import unicode_literals
+from ..helpers import assert_equal, fixture, assert_true, assert_in
+from subprocess import Popen, PIPE
+
+
+def test_happy_path_for_tagging():
```
Comment (nickstenning) [inline test/integration/test_commandline.py]: `test_tagging`...?

File: test/unit/test_fields.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    assert_equal(next(out), {'@message': 'two'})
+
+
+def test_single_field():
+    data = [{'@message': 'one'},
+            {'@message': 'two'}]
+    out = fields(data, ['foobar=baz'])
+
+    assert_equal(next(out), {'@message': 'one',
+                              '@foobar': 'baz'})
```
Comment (nickstenning) [inline test/unit/test_fields.py]: Minor indentation error here (and below) =).

**Comment by nickstenning** [comment]
As discussed in person, I think this should probably be called "keys" (or something other than  "fields"), as logstash's json event format already defines a "fields" key, which corresponds to only a subset of what you're allowing here.

``` json
{
  "@timestamp": "2010-12-21T21:48:33.309258Z",
  "@tags": [ "array", "of", "tags" ],
  "@type": "string",
  "@source": "source of the event, usually a URL."
  "@fields":{
    # a set of fields for this event
    "user": "jordan",
    "command": "shutdown -r":
  } 
  "@message": "the original plain-text message"
}
```

**Comment by nickstenning** [comment]
Merged in 5997eaa.

**Pull request closed without merging**